### PR TITLE
Kk/eswarasai/feature/web3 v2

### PR DESCRIPTION
**Pull request by krzkaczor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ You will also need to install a desired target for example `typechain-target-eth
 | [`typechain-target-ethers`](/packages/typechain-target-ethers)   | [![npm](https://img.shields.io/npm/v/typechain-target-ethers.svg)](https://www.npmjs.com/package/typechain-target-ethers)   | Ethers support             |
 | [`typechain-target-truffle`](/packages/typechain-target-truffle) | [![npm](https://img.shields.io/npm/v/typechain-target-truffle.svg)](https://www.npmjs.com/package/typechain-target-truffle) | Truffle support            |
 | [`typechain-target-web3-v1`](/packages/typechain-target-web3-v1) | [![npm](https://img.shields.io/npm/v/typechain-target-web3-v1.svg)](https://www.npmjs.com/package/typechain-target-web3-v1) | Web3 version 1.x.x support |
+| [`typechain-target-web3-v1`](/packages/typechain-target-web3-v2) | [![npm](https://img.shields.io/npm/v/typechain-target-web3-v2.svg)](https://www.npmjs.com/package/typechain-target-web3-v2) | Web3 version 2.x.x support |
```
Comment (eswarasai) [inline README.md]: @krzkaczor - This must be `[typechain-target-web3-v2]` instead of `v1` 😅

Comment (eswarasai) [reply README.md]: Also, can add `web3-v2` in the `CLI` section under [`README.md`](https://github.com/ethereum-ts/TypeChain#cli)

Comment (eswarasai) [reply README.md]: And maybe [Web3-2.x.x](https://github.com/ethereum-ts/TypeChain#web3-100) under Targets?

Comment (krzkaczor) [reply README.md]: fixd!

Comment (eswarasai) [reply README.md]: Thanks 👍

**Pull request merged**